packages/common/src/operations.graphql

```diff
@@ -57,6 +85,7 @@ query invoice($id: String!) {
             itemName
             numberOfPacks
             packSize
+            locationDescription
```

Comment:
A few of the same thing throughout for location/unit mainly. If we also nest the entity within, we have messed up names - i.e. `location` should be the entity